filters/apimonitoring/doc.go

```diff
@@ -1,35 +1,67 @@
 /*
 
-API Monitoring - apimonitoring() filter
+Package apimonitoring provides filters gathering metrics around API calls
 
-The `apimonitoring` filter adds API related metrics to the monitoring.
+Feature switch & Dependencies
 
-	a: Path("/this-is-monitored")
+This feature is considered experimental and should only be activated explicitly. To
+enable it, the flag `-enable-apimonitoring` must be set when Skipper is launched. Also,
+it does not make sense if none of the metrics implementation is enabled. To use this
+filter, start skipper enabling both. Per instance:
+
+	skipper -enable-apimonitoring -enable-prometheus-metrics
+
+This will enable the API monitoring filter through Prometheus metrics.
+
+
+Configuration
+
+Due to its structured configuration, the filter accepts one parameter of type string
+containing a JSON object.
+
+Details can be found at https://opensource.zalando.com/skipper/reference/filters/#apimonitoring
+(or in this project, under `docs/reference/filters.md`).
+
+Example:
+
+	Path("/this-is-monitored")
 		-> apimonitoring(`{
-			"application_id": "my_app",
-			"path_templates": [
-				"foo/orders",
-				"foo/orders/:order-id",
-				"foo/orders/:order-id/order-items/{order-item-id}"
-				"/foo/customers/",
-				"/foo/customers/{customer-id}/"
+			"apis": [
+				{
+					"application_id": "my_app",
+					"api_id": "orders_api",
+				"path_templates": [
+						"foo/orders",
+						"foo/orders/:order-id",
+						"foo/orders/:order-id/order_item/{order-item-id}"
+					]
+				},
+				{
+					"application_id": "my_app",
+					"api_id": "customers_api",
+					"path_templates": [
+						"/foo/customers/",
+						"/foo/customers/{customer-id}/"
+					]
+				}
 			]
 		}`)
 		-> "https://example.org/";
 
 
-
 Development Helpers
 
-Command line for executing locally:
+The spec and filter log detailed operation information at `DEBUG` level. The -application-log-level=DEBUG switch
+is desirable for debugging usage of the filter.
+
+Command line example for executing locally:
 
-make skipper && \
-  ./bin/skipper \
-    -routes-file "~/temp/test.eskip" \
-    -enable-apimonitoring \
-    -apimonitoring-verbose \
-    -enable-prometheus-metrics \
-    -histogram-metric-buckets=".01,.025,.05,.075,.1,.2,.3,.4,.5,.75,1,2,3,4,5,7,10,15,20,30,60,120,300,600"
+	make skipper && ./bin/skipper \
+		-routes-file "/Users/foo/temp/test.eskip" \
```

Comment:
`~` or `$USER` is more user friendly, than a mac only path.

Reply:
You are totally right! Changed.